src/__tests__/APIService.test.js

```diff
@@ -1,52 +1,22 @@
 const request = require('supertest')
 const APIService = require('../APIService')
 const API = require('../node.js')
+const { user1, user2, user3, notUser } = require('./test-data/users')
 
-jest.setTimeout(30000);
-
-// TODO create data for three users
-// MOCK HTTP TO ADDRESS SERVER
-jest.mock('axios', () => {
-  return {
-    get: jest.fn((url) => {
-      if (url.includes('0x1ee6ae029c6d99ff3a810cf8eaa31d193c89ec9c')) {
-      // https://beta.3box.io/address-server/odbAddress/0x1ee6ae029c6d99ff3a810cf8eaa31d193c89ec9c
-        return { data: {"status":"success",
-                  "data":{"rootStoreAddress":"/orbitdb/QmUGS97iEKTdYKk9eRAeEDcYddjf4cPvEBN3VYWAt4UQzq/122005848b73823cd14509c582c552ca7a4e4e668a7228dd8118cfddf2f7cf8cc21c.root",
-                                       "did":"did:muport:QmQAnachTJXMVHKa5Nu3mZNn5jGrJ9TvHJde3NSp8J4qzL"}}}
-      }
-      return {"status":"error","message":"address not linked"}
-    }),
-    post: () => {
-      // TODO return multiple addresses
-      return { data: {"status":"success",
-                "data":{"rootStoreAddresses":
-                  { '0x1ee6ae029c6d99ff3a810cf8eaa31d193c89ec9c': "/orbitdb/QmUGS97iEKTdYKk9eRAeEDcYddjf4cPvEBN3VYWAt4UQzq/122005848b73823cd14509c582c552ca7a4e4e668a7228dd8118cfddf2f7cf8cc21c.root"}
-                }}}
-        }
-
-  }
-})
+jest.mock('axios', () => require('./mocks/addressServer'))
+jest.mock('redis', () => require('./mocks/orbitRedisCache'))
 
-// MOCK ORBIT DB REDIS CACHE
-jest.mock('redis', () => {
-  const Redis = require('ioredis-mock');
-  const orbitCacheMockData = require('./test-data/orbit-cache')
-  const redis = new Redis({
-    data: orbitCacheMockData
-  })
-  return {
-    createClient: () => redis
-  }
-})
+// posts by 3 users, first two entry deleted
+const openThreadAddress = '/orbitdb/zdpuB18U3dEMq4G4hgShc9Nd1rJM1ZB1JWhsAG8CkmpFHxA6u/3box.thread.spaceone.open'
+// closed to user 1 and user 2, first entry deleted
+const closedThreadAddress = '/orbitdb/zdpuAnq4FhPsq8iMzy4HX7LKUV8MbNjb8oRmk9ghjdhGdRtQJ/3box.thread.spaceone.closedu1u2'
```

Comment:
Are these premade threads?

Reply:
Yeah, some context on the tests, they are more integration style tests, test accounts were used to create some data, spaces, threads, etc, but none of that is mocked, it all in blocks in ipfs test data. And yeah these are two threads created. Thats why all the requests are just snapshot tested instead

Reply:
Makes sense 👍